### DISCUSSION TITLE
Visualising Peridynamics Simulation Results

### DISCUSSION THREAD

**Discussion by ABallisat** [idaholab/moose#16496] [Q&A Tools] [answered]
Hello all,

I am starting to use the Peridynamics module and was wondering what tools people use for visualising the results? I have tried Paraview and Peacock but they show the mesh as a connected grid as in the attached image. I was hoping to be able see the individual particles, as in the image on the Peridynamics module main page. I am probably missing something obvious but any help greatly appreciated!

Alex

![image](https://user-images.githubusercontent.com/40231544/102233745-7e771400-3ee8-11eb-9089-b321030f4100.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

People use paraview and peacock for visualization mostly. 
To achieve an image like the peridynamics module one, you can use the 'points' in paraview. They will be colored based on the value of the function visualized. And there size can be customized. 
I think that image is still showing the mesh, but restricted to the solid phase, and deformed by creep and all the other phenomena from irradiation. 

Guillaume
